poetry.lock

```diff
@@ -286,7 +286,7 @@ format_nongpl = ["idna", "jsonpointer (>1.13)", "webcolors", "rfc3986-validator
 
 [[package]]
 name = "junitparser"
-version = "2.0.0"
+version = "1.6.3"
```

Comment:
I think it should be added to `pyproject.toml` with hardcoded value to `1.6.3` If we don't do that it will update version after `poetry update` or any operation which involves dependency resolving

Reply:
An update made by @kkania-splunk resolves this issue with the commit here: https://github.com/splunk/pytest-splunk-addon/pull/275